# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: modules/libpref/init/StaticPrefList.yaml
# Commit: 0d746825f92a
# Full Hash: 0d746825f92a3039433aa275997c6e7f2ebfaf56
# Author: Jonathan Kew <jkew@mozilla.com>
# Date: 2021-05-18 21:36:28
# Regressor Bug: 1708768
# File Overlap Count: 2
# Description:
#   Bug 1708768 - Make InitFontList safe to call off-main-thread on macOS, and run font-list initialization on a separate thread during startup. r=lsalzman
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D115295
# ==============================================================================

diff -r fbe7310ee5f3 -r 0d746825f92a modules/libpref/init/StaticPrefList.yaml
--- a/modules/libpref/init/StaticPrefList.yaml	Tue May 18 11:10:43 2021 +0000
+++ b/modules/libpref/init/StaticPrefList.yaml	Tue May 18 11:10:43 2021 +0000
@@ -4617,7 +4617,7 @@
   value: true
   mirror: once
 
-- name: gfx.font-list.lazy-init.enabled
+- name: gfx.font-list-omt.enabled
   type: bool
 #if defined(XP_MACOSX)
   value: true
